xhr/abort-during-upload.htm

```diff
@@ -11,7 +11,7 @@
   <body>
     <div id="log"></div>
     <script>
-      var test = async_test(document.title, {timeout:1100})
+      var test = async_test(document.title);
```

Comment:
```suggestion
      var test = async_test()
```

Reply:
`document.title` is the default test name if none is given, so this can be removed.